src/components/IconButton/index.jsx

```diff
@@ -0,0 +1,58 @@
+import cls from 'classnames';
+import React, { useCallback } from 'react';
+import styles from './styles.module.css';
+import {
+  flex, effects, typography as type, margin,
+} from '../../theme';
+import BaseIcon from '../BaseIcon';
+
+/**
+ * Render a button with a full-sized icon and label
+ * @param {Object} props
+ * @param {Boolean} [props.disabled] - Set disabled state
+ * @param {String} [props.className] - Extra classes
+ * @param {Function} [props.onClick] - onClick event listener
+ * @param {Boolean} [props.setWhite] - Set to `true` to use the
+ * alternate white version
+ * @param {String} props.iconName - Hero icon component name. All names end in `Icon` (e.g. `"GlobeAltIcon"`) {@link https://heroicons.com/}
+ * @param {String} props.label - Label of button below icon
+ * @returns {React.ReactElement}
+ */
+export default function IconButton(props) {
+  const {
+    disabled = false,
+    className = '',
+    onClick = () => null,
+    iconName,
+    label,
+    setWhite = false,
+  } = props;
+
+  const btnClass = cls(
+    styles.btn,
+    setWhite && styles.white,
```

Comment:
So classnames actually let's you do something like

```
{ [styles.white]: setWhite } 
```

which definitely looks silly as one of a number of list, but generally I just make the last argument the "optional" classes argument which is an object like that above. Makes it easier to quicly read, what are styles that are always applied, and which are conditional.